open_access/cds_imaging.md

```diff
@@ -0,0 +1,41 @@
+---
+order: 995
+---
+
+# Accessing Images via Cancer Data Service (CDS)
+
+HTAN Imaging Level 2 data is now available through the [NCI Cancer Data Service (CDS)](https://datacommons.cancer.gov/repository/cancer-data-service)
+
+Data access via Seven Bridges Cancer Genomics Cloud (SB-CGC) requires a CGC account [[register here](https://docs.cancergenomicscloud.org/docs/sign-up-for-the-cgc)]. For further information on using SB-CGC resources including programmatic access options, you can explore their [online documentation](https://docs.cancergenomicscloud.org/docs).
```

Comment:
Just checking on this:  If I register for a SB-CGC account, but I don't have dbGap access to HTAN, can one still download images?

Reply:
tx!